tgui/packages/tgui/interfaces/NtosMain.js

```diff
@@ -29,21 +29,39 @@ export const NtosMain = (props, context) => {
       width={400}
       height={500}>
       <NtosWindow.Content scrollable>
-        {!!has_light && (
+        {(!!has_light || !!removable_media.length) && (
```

Comment:
Would be more readable if you combined it like this:
```suggestion
        {Boolean(has_light || removable_media.length) && (
```